server/gfmd/db_pgsql.c

```diff
@@ -4255,6 +4255,141 @@ gfarm_pgsql_mdhost_load(void *closure,
 	return (GFARM_ERR_NO_ERROR);
 }
 
+
```

Comment:
move the user_auth related functions to just after the gfarm_pgsql_user_load() function,
to match the modified order of struct db_ops.

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not applied above change. Could you change it?